apps/sms/views/conversation/js/attachment_renderer.js

```diff
@@ -203,6 +203,16 @@
 
               // It's essential to remember real image URL to revoke it later
               attachmentContainer.dataset.thumbnail = data.thumbnail.url;
+
+              var fileNode = contentContainer.querySelector('.file-name');
+              if (this._attachment.name) {
+                fileNode.removeAttribute('data-l10n-id');
+                fileNode.textContent = this._attachment.name.slice(
+                  this._attachment.name.lastIndexOf('/') + 1
```

Comment:
`this` won't work here because the function is not bound.

You can change the anonymous functions to arrow functions so that the outside context is kept